_bookdown.yml

```diff
@@ -68,6 +68,7 @@ rmd_files: [
   "new_pages/combination_analysis.Rmd",
   "new_pages/transmission_chains.Rmd",
   "new_pages/phylogenetic_trees.Rmd",
+  "new_pages/AMR.Rmd",
```

Comment:
In the mail you advised to put it below Phylogenetic Trees in the Analysis section. But Phylogenetic Trees is not in the Analysis section, it's in the Data Visualization section. So shall we put AMR in the Analysis section, or do want to keep it here? Both fine to me, although I consider this chapter to be more about analysis than about datavis.

Reply:
Ah you are right! I misread. Agree analysis section is best. For now let's just put it at the end of the Analysis section.
Thanks for setting up this PR!